packages/teams-js/test/public/app.spec.ts

```diff
@@ -417,91 +423,96 @@ describe('AppSDK-app', () => {
   });
 
   describe('navigateCrossDomain', () => {
```

Comment:
I noticed that this test, which you did not add, seems like it should actually belong in the pages.spec.ts file, not the app.spec.ts. I suspect this was an oversight when the APIs were organized into the pages capability. Will follow up offline about whether these should be moved as part of this change.